### ISSUE TITLE
never works! Throws Cannot use strict priority all the time

### ISSUE THREAD

**Issue by antonkulaga** [QuantStack/mamba#86]
I did not manage to install a single package with mamba!
For instance If I say:
```
mamba install -c conda-forge dvc
```
I get:
```

Looking for: ['dvc']

Cannot use strict priority with mamba!
```
And ALL the packages I tried to install give the same error.
My OS is latest Linux Mint. I use the mamba version 0.1.0 from conda-forge
I also have flexible channel priorities so it is strange why it complains about strict priority

**Comment by antonkulaga** [comment]
Actually, it seems I indeed had "strict", I simple confisued --describe with --show in conda config when I checked priority values. In the same time it will still be important to make strict working

**Comment by wolfv** [comment]
Hi Anton, thanks for your bug report. We do have strict priority working on the master branch. We'll make a release shortly I hope! 

Wolf

**Comment by wolfv** [comment]
this is fixed in the latest release, available from conda-forge shortly :) (version 0.1.1).
